src/StatsBase.jl

```diff
@@ -21,7 +21,7 @@ import Statistics: mean, mean!, var, varm, varm!, std, stdm, cov, covm,
                    cor, corm, cov2cor!, unscaled_covzm, quantile, sqrt!,
                    median, middle
 using StatsAPI: StatisticalModel, RegressionModel
-import StatsAPI: pairwise, pairwise!, params, params!,
```

Comment:
why `params` is not tested?

Reply:
It's not actually (and has never been) reexported, so if we tested something it would be that it's *not* defined. :-)

Reply:
Woops, actually the fact that we don't export it doesn't mean it's fine to stop importing it. Packages probably rely on StatsBase defining it. It was added by https://github.com/JuliaStats/StatsBase.jl/pull/274. I've added a commit to fix this.